### ISSUE TITLE
Show value instead of curency amount

### ISSUE THREAD

**Issue by nymeia** [mlbonniec/MMM-Coinbase#17]
Would it be possible to set i.e. XLM   35,52 EU, instead of 124,2530943 XLM?
I want to see how much my currency is worth, not the amount of coins I got.
Or both would be great.

**Comment by WeeskyBDW** [comment]
Show less numbers after comma ?

**Comment by nymeia** [comment]
> Show less number after comma ?

No, i want to see how much money my coins are worth, not the amount of coins that i have.

So instead of: BTC      1.0000000
I want to see: BTC       20.000,00EUR

or 

XLM €34,87 or 34,87EUR
instead of XLM 124,2530943

**Comment by WeeskyBDW** [comment]
Oh ok show converted value. Idk if it's possible directly with the lib used by MMC-coinbase but still possible with externals requests

**Comment by Lambenttelos** [comment]
I found a way to do this. I may see about adding an option to the config and and submitting a pull request.

edit `./modules/MMM-Coinbase/MMM-Coinbase.js`

and replace:
`columnAmountElement.innerHTML = "<span>" + accts.balance.amount + "</span>";`

with:
`columnAmountElement.innerHTML = "<span>" + accts.native_balance.amount + " " + accts.native_balance.currency  + "</span>";`
